cmd/localstack/main.go

```diff
@@ -20,6 +20,7 @@ type LsOpts struct {
 	User                string
 	CodeArchives        string
 	HotReloadingPaths   []string
+	FileWatcher         string
```

Comment:
```suggestion
	FileWatchingStrategy         string
```

kinda nit/suggestion, feel free to ignore

Reply:
I like it, was not happy with the name